### PR TITLE
Created basic flask project structure

### PR DESCRIPTION
**Pull request by lematia1-accord1**
I have added app.py which contains the main flask file

### CODE REVIEW

File: app.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+# init app
+app = Flask(__name__)
+# create a route for drone_service
+@app.route("/drone-service", methods = ["GET"])
+def get_message_from_drone():
+    return jsonify({"message": "hellow"})
+# run the server
+if __name__ == "__main__":
+    app.run(debug=True)
```
Comment (egimaben) [inline app.py]: please make sure there's a new line at end of file

File: app.py [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# import the required packages
```
Comment (egimaben) [inline app.py]: we no longer need this comment, plus all others, the code should be self-documenting

File: app.py [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+from flask import Flask, request, jsonify
+from flask_sqlalchemy import SQLAlchemy
```
Comment (egimaben) [inline app.py]: unused import, why do we have it here when we are not using it

**Review by egimaben — CHANGES_REQUESTED** [review]
the structure looks good, but i left some comments for you for now

**Comment by egimaben** [comment]
please add tests for your code as well

File: app.py [hunk 4]
```diff
@@ -0,0 +1,12 @@
+
+from flask import Flask, request, jsonify
+from flask_marshmallow import Marshmallow
```
Comment (egimaben) [inline app.py]: It's not clear why we need this import here.

File: app.py [hunk 5]
```diff
@@ -0,0 +1,11 @@
+
+from flask import Flask, request, jsonify
+#from flask_marshmallow import Marshmallow
```
Comment (egimaben) [inline app.py]: you just commented out the import, commented code is confusing in a code base, better to just remove it. If someone reads your code, they will waste time wondering why the import and why is it commented, so we should just remove it

**Comment by egimaben** [comment]
> please add tests for your code as well

we can ignore this for now and tests in a follow-up PR

File: app.py [hunk 6]
```diff
@@ -0,0 +1,9 @@
+from flask import Flask, request, jsonify
```
Comment (egimaben) [inline app.py:1]: leave a line to separate the imports from the rest of the code so that it looks clean

File: app.py [hunk 7]
```diff
@@ -0,0 +1,9 @@
...
+app = Flask(__name__)
+@app.route("/drone-service", methods = ["GET"])
```
Comment (egimaben) [inline app.py:5]: also leave a line between the app instantiation and the endpoints.. this will make the code look more organized

**Review by egimaben — CHANGES_REQUESTED** [review]

**Comment by lematia1-accord1** [comment]
@egimaben , i have attended all the comments ,kindly advice .

**Review by egimaben — APPROVED** [review]
looks good

**Comment by egimaben** [comment]
@lematia1-accord1 thanks for the changes, LGTM

**Pull request merged**